pixi.toml

```diff
@@ -79,6 +79,25 @@ rerun-web-release = { cmd = "cargo run --package rerun-cli --no-default-features
 # (this looks heavy but takes typically below 0.1s!)
 rerun-build-web-release = "rustup target add wasm32-unknown-unknown && cargo run -p re_build_web_viewer -- --release -g"
 
+rs-check = "rustup target add wasm32-unknown-unknown && python scripts/ci/rust_checks.py"
+
+# Note: running just `cargo deny check` without a `--target` can result in
+# false positives due to https://github.com/EmbarkStudios/cargo-deny/issues/324
+rs-cargo-deny = '''
```

Comment:
This seems worth writing as a python script that we invoke.

Reply:
could also fold that into rust_checks.py hmmmm

Reply:
did that now. lessens parallism but removes some overhead as well, nice to have all cargo checks in one place imho